### PR TITLE
Configure NPM module

### PR DESCRIPTION
**Pull request by jhbertra**
Adds configuration for an NPM module. Also adds a standard webpack config.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -35,9 +35,169 @@ You will then be able to install any package on the latest psc-package set, as
 well as `json-helpers`, `servant-support`, `web-common`, and `markdown` via
 `spago install`.
 
-=== How to depend on the static sources in this project
+=== How to override this dependency locally
```
Comment (hrajchert) [inline README.adoc:38]: Thanks for this 🙌

File: flake.nix [hunk 2]
```diff
@@ -67,6 +67,10 @@
               excludes = [ ".*spago-packages.nix$" ];
             };
             inherit purs-tidy-hook;
+            prettier = {
+              enable = true;
+              types_or = [ "javascript" "css" "html" ];
+            };
```
Comment (hrajchert) [inline flake.nix:73]: is this applied to the old JS files or just new commits? we might want to do a separate PR in which we apply prettier to old files.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,53 @@
...
+    "chartist-plugin-axistitle": "^0.0.7",
+    "chartist-plugin-tooltips": "^0.0.17",
+    "clean-webpack-plugin": "^4.0.0",
+    "css-loader": "^6.5.1",
+    "json-bigint": "^1.0.0",
+    "mini-css-extract-plugin": "^2.4.4",
+    "postcss-loader": "^6.2.0"
+  },
+  "devDependencies": {
+    "@popperjs/core": "^2.9.2",
```
Comment (hrajchert) [inline package.json:38]: this is repeated in peerDependencies

File: flake.nix (hunk 2 above)
Comment (hrajchert) [reply flake.nix:73]: nevermind... just saw the changes

**Review by hrajchert — APPROVED** [review]
LGTM

File: flake.nix (hunk 2 above)
Comment (jhbertra) [reply flake.nix:73]: Few enough files in this repo that I don't think it's a huge deal.  I'm going to do dedicated formatting PRs for the other repos.

File: package.json (hunk 3 above)
Comment (jhbertra) [reply package.json:38]: Yes, as it should be. It's in `devDependencies` here because we don't want it installed when this package is `npm install;d` (It's just used in tests / for local development). But the package that installs web-common is required to install this as a peer dependency. This is all because of the fact that we install this package via `spago` and not `npm` for the purescript dependencies... it's unfortunately rather complicated.
